scripts/sync-hf-model-repos.sh

```
@@ -13,31 +13,50 @@
   [unirig]="unirig/encoder.onnx unirig/decoder.onnx unirig/embed.onnx"
   [skintokens]="skintokens/skintokens.json skintokens/mesh_cond.onnx skintokens/vae_cond.onnx skintokens/embed.onnx skintokens/skin_decode.onnx skintokens/decoder.onnx skintokens/decoder.onnx.data"
   [triposr]="triposr/triposr_encoder.onnx triposr/triposr_encoder_int8.onnx triposr/triposr_decoder.onnx"
   [triposg]="triposg/triposg_image_encoder.onnx triposg/triposg_dit_step.onnx triposg/triposg_dit_step.onnx.data triposg/triposg_vae_latents.onnx triposg/triposg_vae_decoder.onnx"
+  [u2net]="rembg/u2net.onnx"
+  [smolvlm]="caption/SmolVLM-500M-Instruct-Q8_0.gguf caption/mmproj-SmolVLM-500M-Instruct-Q8_0.gguf"
+  [motion]="motion/motion-library.json"
 )
 
 sync_one() {
-  local model=$1 repo="$OWNER/QtMeshEditor-$1-onnx"
+  local model=$1 repo="$OWNER/${REPOS[$model]}"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)